modules/system/applications.nix

```diff
@@ -22,14 +22,26 @@ in
 
     system.activationScripts.applications.text = ''
       # Set up applications.
-      echo "setting up ~/Applications..." >&2
+      echo "setting up /Applications/Nix Apps..." >&2
+
+      ourLink () {
+        local link
+        link=$(readlink "$1")
+        [ -L "$1" ] && [ "''${link#*-}" = 'system-applications/Applications' ]
```

Comment:
Not sure this is the best condition, but not sure either what would be given that a link to the store isn't enough. Given that the name now has significance a comment in the corresponding expression would be a good idea.

Reply:
Which name do you mean, the suffix `system-applications/Applications`, in case someone happens to create a link at `~/Applications` for a (store) path ending with that suffix or something?